tool/src/main/java/org/wildfly/security/tool/MaskCommand.java

```diff
@@ -80,23 +84,25 @@ public void execute(String[] args) throws Exception {
 
         String salt = cmdLine.getOptionValue(SALT_PARAM);
         if (salt == null) {
-            setStatus(GENERAL_CONFIGURATION_ERROR);
-            throw ElytronToolMessages.msg.saltNotSpecified();
+            salt = generateSecureRandomSalt(8);
```

Comment:
I think there's a `generateRandomSalt` method in `org.wildfly.security.password.impl.PasswordUtil`. I know it won't be possible to use that directly here but the overall approach used there could also be used here I think.

Reply:
Hi @fjuma , thanks for your review, I have updated the generate method.